man/profile.doc

```diff
@@ -144,12 +174,12 @@ While the program executes under the profiler, the system builds a
 kernel: one that starts a new goal (\emph{profCall}), one that tells the
 system which goal is resumed after an \emph{exit} (\emph{profExit}) and
 one that tells the system which goal is resumed after a \emph{fail}
-(i.e., which goal is used to \emph{retry} (\emph{profRedo})). The
-profCall() function finds or creates the subnode for the argument
+(i.e., which goal is used to \emph{retry} (\emph{profFail})).
```

Comment:
use \exam{...} for examples (not \emph{...})

Reply:
`\const{}` is for constants (identifiers, atoms, ...), \exam{} is for (example) inline code fragments (i.e., generic "code").  Functions are written `name()` (will be styled automatically).    Terms may be written as `\term{name}{args}`, especially if the arguments are placeholders.

Reply:
Thanks for proof reading; I'll update the PR to fix the typos.

Regarding "classic" mode; perhaps I'm overly pedantic but I think a mode which displays incorrect information in some situations is a mistake. It's (too?) easy to enable that mode buy just by omitting the `ports` option so, personally, I'd rather not legitimize it further in the doc.

As for the tex conventions used by SWIP, I just tried to mimic what was there previously as I don't really understand the nuances, and the changes required (if any) to what's been submitted. This is just a draft so I assume the editor-in-chief will have the final say for the official version.

Reply:
I'm still confused: if I specify `ports(false)`, do I get the current (somewhat wrong) behaviour? Do I get a different behaviour if I don't specify either `ports(false)` or `ports(true)`?

If `ports(false)` gives the same behaviour as not specifying the `ports` option at all, then there's no problem. If this isn't the case, I don't like it because the API would have no way of specifying the default behaviour. 

[I'll leave the markup fixing to Jan -- I often get it wrong because there's no documentation about the markup and I got lost while trying to read the TeX macros)

Reply:
There are three different behaviours at the moment. The API (predicate `'$profile'/4`) supports all three. The user doc "should" explain how each one is specified in the `profile/2` predicate (no `ports` option defaults to "classic" mode).

I'd rather there were just two modes: `ports(true)` and `ports(false)`; that's how I intend to use it.